### ISSUE TITLE
Can i know the training environment?

### ISSUE THREAD

**Issue by seongkyun** [ShuangXieIrene/ssds.pytorch#39]
I wanna train and get weight file of rfb mobilenet ssd model end-to-end by myself. 
Should I need to load pretrained feature extractors weights? Then how can i get that weight file?
Can i know the training environment like epoch, learning rate, scheduler and learning decaying parameters?

**Comment by lupotto** [comment]
Hi @seongkyun,

> Should I need to load pretrained feature extractors weights?  Then how can i get that weight file?

You can either train from scratch or with pre-trained weights. 

> Then how can i get that weight file?

The weights are located on the main page of the repository, need to click on the accuracy (in blue). 

> Can i know the training environment like epoch, learning rate, scheduler and learning decaying parameters?

You can specify all this parameters in the cfg file.

Cheers!

**Comment by seongkyun** [comment]
Thank you, @lupotto .
I have another question.
Then, the training specifications in cfg file is for end-to-end(from scratch) training or with pre-trained backbone network training?

I mean, if i train my net from scratch with only cfg training specifications, then can I get same accuracy on main page results?

**Comment by lupotto** [comment]
> Then, the training specifications in cfg file is for end-to-end(from scratch) training or with pre-trained backbone network training?

What I know is that the weights that they provide are end-to-end. If they trained with pre-trained weights or not I don't know, you have to ask them.

> I mean, if i train net by myself from scratch with only cfg training specifications, then can I get same accuracy on main page results?

I am not 100% sure because I trained from scratch with my own dataset so I didn't check if the results correspond to the numbers that the repository claim.
